.swcrc

```diff
@@ -0,0 +1,21 @@
+{
+  "jsc": {
+    "parser": {
+      "syntax": "typescript",
+      "tsx": true
+    },
+    "transform": {
+      "react": {
+        "runtime": "automatic"
+      }
+    }
+  },
+  "env": {
+    "targets": [
```

Comment:
Currently there's an issue with [browserslist](https://github.com/browserslist/browserslist) support so we set it directly in here